src/plugins/orangehrmClaimPlugin/test/fixtures/ClaimAttachment.yaml

```diff
@@ -32,14 +44,20 @@ CurrencyType:
-  2: { id: 3, emp_number: 2, added_by: 3, reference_id: 202301170000003, event_type_id: 4, description: "Equipment purchase for IT department", currency_id: USD, is_deleted: false, status: 'INITIATED', created_date: "2022-10-15 14:00:00", submitted_date: "2022-10-15 14:00:00" },
-  3: { id: 4, emp_number: 1, added_by: 2, reference_id: 202301170000004, event_type_id: 2, description: "Salary expenses for temporary employee", currency_id: USD, is_deleted: false, status: 'INITIATED', created_date: "2022-09-01 08:00:00", submitted_date: "2022-09-30 18:00:00" },
-  4: { id: 5, emp_number: 4, added_by: 1, reference_id: 202301170000005, event_type_id: 3, description: "Expenses for social media advertising campaign", currency_id: CAD, is_deleted: false, status: 'INITIATED', created_date: "2022-08-15 12:00:00", submitted_date: "2022-08-20 16:00:00" },
-  5: { id: 6, emp_number: 2, added_by: 3, reference_id: 202301170000006, event_type_id: 4, description: "Training expenses for team building seminar", currency_id: USD, is_deleted: true, status: 'INITIATED', created_date: "2022-07-01 09:00:00", submitted_date: "2022-07-15 17:00:00" }
+  1: { id: 2, emp_number: 1, added_by: 2, reference_id: 202301170000002, event_type_id: 1, description: "Marketing expenses for trade show", currency_id: CAD, is_deleted: false, status: 'REJECTED', created_date: "2022-11-20 09:00:00", submitted_date: "2022-11-25 15:00:00" },
+  2: { id: 3, emp_number: 1, added_by: 2, reference_id: 202301170000003, event_type_id: 4, description: "Equipment purchase for IT department", currency_id: USD, is_deleted: false, status: 'SUBMITTED', created_date: "2022-10-15 14:00:00", submitted_date: "2022-10-15 14:00:00" },
+  3: { id: 4, emp_number: 4, added_by: 1, reference_id: 202301170000004, event_type_id: 2, description: "Salary expenses for temporary employee", currency_id: USD, is_deleted: false, status: 'SUBMITTED', created_date: "2022-09-01 08:00:00", submitted_date: "2022-09-30 18:00:00" },
+  4: { id: 5, emp_number: 4, added_by: 1, reference_id: 202301170000005, event_type_id: 3, description: "Expenses for social media advertising campaign", currency_id: CAD, is_deleted: false, status: 'REJECTED', created_date: "2022-08-15 12:00:00", submitted_date: "2022-08-20 16:00:00" },
+  5: { id: 6, emp_number: 1, added_by: 1, reference_id: 202301170000006, event_type_id: 4, description: "Training expenses for team building seminar", currency_id: USD, is_deleted: false, status: 'INITIATED', created_date: "2022-07-01 09:00:00", submitted_date: "2022-07-15 17:00:00" }
+  6: { id: 7, emp_number: 1, added_by: 1, reference_id: 202301170000007, event_type_id: 4, description: "Training expenses for team building seminar", currency_id: USD, is_deleted: false, status: 'REJECTED', created_date: "2022-07-01 09:00:00", submitted_date: "2022-07-15 17:00:00" }
 
 ClaimAttachment:
```

Comment:
in attachment table `eattach_attachment` column we are storing data as blobs, not as base64 encoded